probe/endpoint/connection_tracker.go

```diff
@@ -109,6 +111,24 @@ func (t *connectionTracker) performFlowWalk(rpt *report.Report, seenTuples map[s
 		seenTuples[tuple.key()] = tuple
 		t.addConnection(rpt, tuple, "", extraNodeInfo, extraNodeInfo)
 	})
+	return seenTuples
+}
+
+func (t *connectionTracker) existingFlows() map[string]fourTuple {
+	seenTuples := map[string]fourTuple{}
+	if !t.conf.UseConntrack {
+		log.Warnf("Not using conntrack: disabled")
```

Comment:
Fair enough. Will remove.

It turns out this change actually fixed a logic bug too: Previously, it would run conntrack here even when the flag wasn't set - all the flag did was suppress the "not supported by the kernel" warning.